integration/testdata/nodejs_simple_app/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Even if we move back to having a buildpack_integration_test.go at the top level, files like this for the test app could stay under the integration directory.